spanner/spec/database_leader_placement_samples_spec.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../database_leader_placement_samples"
+require_relative "./spec_helper"
+
+describe "Spanner database leader placement" do
+  before :each do
+    cleanup_database_resources
+  end
+
+  after :each do
+    cleanup_database_resources
+  end
+
+  example "instance_config" do
+    capture do
+      instance_config project_id: @project_id,
+                      instance_config_id: "cloud-devel-global-config"
```

Comment:
this should be an instance config in prod instead of a staging one.

Reply:
done